napari/_qt/layers/qt_points_layer.py

```diff
@@ -173,21 +173,17 @@ def _on_size_change(self, event):
             self.sizeSlider.setValue(int(value))
 
     def _on_edge_color_change(self, event):
+        color = self.layer.current_edge_color
         with self.layer.events.edge_color.blocker():
-            index = self.edgeComboBox.findText(
-                self.layer.current_edge_color, Qt.MatchFixedString
-            )
+            index = self.edgeComboBox.findText(color, Qt.MatchFixedString)
```

Comment:
What do you think about adding the name/hex string and selecting it in the combobox if it isn't found in the list? I am a bit torn because the user can currently just select the point with the desired color to "pick" it even if it isn't directly selectable from the list. However, it might be nice to have something displayed in the box.

If you do want to add this behavior, I believe findText() returns -1 when the entry isn't found. I think something like this would work

```python
        with self.layer.events.edge_color.blocker():
            index = self.edgeComboBox.findText(color, Qt.MatchFixedStr)
            if index == -1:
                self.edgeComboBox.addItem(color)
                # I think it will always get added to end - could also use findText() again
                index = self.edgeComboBox.count()

Reply:
I like the idea of displaying the hex string if the color is not found